app/models/dog_searcher.rb

```diff
@@ -71,7 +71,9 @@ def self.search(params: {}, manager: false)
   private
 
   def tracking_id_search?
```

Comment:
Also I just learned that `===` when the regex is the first argument returns true / false, good stuff!